src/bridge/views/Login.js

```diff
@@ -0,0 +1,131 @@
+import { Nothing, Just } from 'folktale/maybe';
+import React from 'react';
+import * as azimuth from 'azimuth-js';
+
+import { H1, Row, Col } from '../components/Base';
+import HorizontalSelector from '../components/HorizontalSelector';
+
+import Ticket from './Login/Ticket';
+import Mnemonic from './Login/Mnemonic';
+import Advanced from './Login/Advanced';
+
+import * as need from '../lib/need';
+import { compose } from '../lib/lib';
+import { ROUTE_NAMES } from '../lib/routeNames';
+
+import { withHistory } from '../store/history';
+import { withNetwork } from '../store/network';
+import { withWallet } from '../store/wallet';
+import { withPointCursor } from '../store/pointCursor';
+
+const TABS = {
+  TICKET: Symbol('TICKET'),
+  MNEMONIC: Symbol('MNEMONIC'),
+  ADVANCED: Symbol('ADVANCED'),
+};
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentTab: TABS.TICKET,
+      wallet: Nothing(),
+    };
+
+    this.handleTabChange = this.handleTabChange.bind(this);
+    this.continue = this.continue.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.setNetworkType(this.props.networkType);
```

Comment:
this line won't actually do anything, since the initial network type is already the same as `props.networkType`

Reply:
You're right. I'm imagining cases though where we return to Login after having been in a different flow that may have used a different network. Very unlikely of course, but it feels nice having this guarantee, instead of needing to think about how Login might interact with other functionality.

Does than sound sane?

Reply:
even then, it wouldn't reset to the initial network type, but only re-set the current network

if we want a "reset to existing network" behavior

1) let's not add that until we actually need it (the figma designs don't have anything about networks, actually - jimmy confirmed that it isn't something they want normal users to toggle)
2) we'd put that within the `_useNetwork` hook itself as a function like `resetNetwork()` that sets the state to the `initialNetwork` that was provided to the hook.

Reply:
_cleans glasses_ Oh, I see what you're saying. This is a straight no-op. (^:

Fair on both points!